src/contracts/pods/EigenPodManager.sol

```diff
@@ -215,6 +222,54 @@ contract EigenPodManager is
         ownerToPod[podOwner].withdrawRestakedBeaconChainETH(destination, shares);
     }
 
+    /// @notice Called by proving service to fulfill partial withdrawal proof requests
+    function proofServiceCallback(
+        WithdrawalCallbackInfo calldata callbackInfo
+    ) external onlyProofService nonReentrant {
+        require(proofServiceEnabled, "EigenPodManager.proofServiceCallback: offchain partial withdrawal proofs are not enabled");
```

Comment:
again seems like this should probably be just another pausing flag to me.

I think in this function is the more appropriate place to check this parameter, as it is at the start of the chain of calls.

Reply:
We check it in _processPartialWithdrawal() which is called in _verifyAndProcessWithdrawal()l, which is in turn called in verifyAndProcessWithdrawals() which is called directly from the pod  - that's why I added it as a separate check there.